application/pm.py

```diff
@@ -116,6 +116,18 @@ def create_project_dir(self):
             progress_msg.setText("Writing configuration files...")
             self._write_to_project_config()
             copy("default/tracking.cfg", os.path.join(pr_path, "tracking.cfg"))
+	    copy("default/classifier.cfg", os.path.join(pr_path, "classifier.cfg"))
```

Comment:
Looks like there's some mismatch with indentation here. Not sure if it's a tabs vs spaces issue or a GitHub being weird thing. Maybe we should decide tabs vs spaces as a group?